Formula/saxon.rb

```diff
@@ -3,7 +3,7 @@ class Saxon < Formula
   homepage "https://saxon.sourceforge.io"
   url "https://github.com/Saxonica/Saxon-HE/blob/main/12/Java/SaxonHE12-0J.zip"
   version "12.0"
-  sha256 "c476746275dd5a0de1d203e89c21a249a02efe33350b560c4086cb08b0816be7"
+  sha256 "32a638c2461d94c3cf68b25658bb475cdfc09c17de36f8b97bbd73bab6e3c4e4"
```

Comment:
For me, `openssl dgst -sha256 SaxonHE12-0J.zip` returns:

```
SHA256(SaxonHE12-0J.zip)= c476746275dd5a0de1d203e89c21a249a02efe33350b560c4086cb08b0816be7
```

I wonder why our calculations differ?

Reply:
I think my original sha256 was correct. [CI is choking on this](https://github.com/Homebrew/homebrew-core/actions/runs/3931752350/jobs/6723515405) and reports it expected my original value:

```
Downloaded to: /Users/brew/Library/Caches/Homebrew/downloads/144052f22bce4622346f65b795a6283e6c7e3c29d4d9665cc880ad5f15e550f1--SaxonHE12-0J.zip
  SHA256: c476746275dd5a0de1d203e89c21a2[49](https://github.com/Homebrew/homebrew-core/actions/runs/3931752350/jobs/6723515405#step:6:50)a02efe333[50](https://github.com/Homebrew/homebrew-core/actions/runs/3931752350/jobs/6723515405#step:6:51)b560c4086cb08b0816be7
  Warning: Formula reports different sha256: 32a638c2461d94c3cf68b25658bb475cdfc09c17de36f8b97bbd73bab6e3c4e4
```